### PR TITLE
(FM-7550) Convert cisco_aaa_authentication_login acceptance test to d…

### PR DESCRIPTION
**Pull request by willmeek**
…ual mode

### CODE REVIEW

**Review by Thomas-Franklin — APPROVED** [review]
👍 nicely done

**Comment by Thomas-Franklin** [comment]
Think this just needs rebased and resolved due to utility lib changes. Happy to merge once it is done 👍

**Comment by willmeek** [comment]
> Think this just needs rebased and resolved due to utility lib changes. Happy to merge once it is done 👍

Rebased :)

File: tests/beaker_tests/lib/utilitylib.rb [hunk 1]
```diff
@@ -339,13 +345,13 @@ def test_manifest(tests, id)
         # if !code.include?($?.exitstatus)
         #  raise 'Errored test'
         # end
-        output = `#{AGENTLESS_COMMAND} --apply #{@temp_agentless_manifest.path}`
+        output = `#{AGENTLESS_COMMAND} --apply #{@temp_agentless_manifest.path} 2>&1`
         unless output.include? 'Applied catalog'
```
Comment (Thomas-Franklin) [inline tests/beaker_tests/lib/utilitylib.rb]: This seems to be causing problems and the tests are failing for the `multiple_auths` tests as the output doe not include `'Applied catalog'`

Comment (willmeek) [reply tests/beaker_tests/lib/utilitylib.rb]: I may have missed something from the rebase merge, let me check

Comment (willmeek) [reply tests/beaker_tests/lib/utilitylib.rb]: image

Yep merge gone bad

I'll fix that up

**Pull request merged**